explorer/src/documentation/Main.hs

```diff
@@ -75,6 +76,9 @@ main = do
 instance HasSwagger api => HasSwagger (MultipartForm a :> api) where
     toSwagger Proxy = toSwagger $ Proxy @api
 
+instance HasSwagger v => HasSwagger (VerbMod mod v) where
+    toSwagger _ = toSwagger $ Proxy @v
```

Comment:
Don't you mind moving it to `Pos.Util.Servant`?
We already have similar instance for wallet, but I couldn't perform this movement earlier, because at that time adding even a single dependency to subproject time-outed CI